### DISCUSSION TITLE
How can i change italic icon to normal one ?

### DISCUSSION THREAD

**Discussion by theaungmyatmoe** [nvim-neo-tree/neo-tree.nvim#614] [Ideas]
See in neo tree file browser. Italic icon are really ugly. btw how can i change italic text to normal one?

![Screenshot from 2022-11-18 05-29-04](https://user-images.githubusercontent.com/65492233/202578269-b08346e1-e52c-4a13-8bed-7dc74033aff2.png)

**Comment by nhat-vo** [comment]
A simple fix will just be setting the highlight group yourself. Here is what I did for example 
```
vim.cmd[[
highlight NeoTreeGitConflict gui=bold
highlight NeoTreeGitUntracked gui=none
highlight NeoTreeRootName gui=bold
]]
```

**Reply by xmalter** [reply]
@nhat-vo Sorry for stupid questions, but where should I place this code to make it work? I'm using Astronvim

**Reply by nhat-vo** [reply]
You can put it at the bottom of your config. For Astronvim, it seems to be `init.lua`.
